### PR TITLE
added heart as an image option

### PR DESCRIPTION
**Pull request by x4m3**
let me know if i did something wrong

i tested and it works.

### CODE REVIEW

**Comment by x4m3** [comment]
forgot to add a screenshot, my bad   
screenshot from 2018-01-20 23-37-07

you can check it out [on my profile](https://github.com/x4m3?tab=overview&from=2014-12-01&to=2014-12-31)

File: gitfiti.py [hunk 1]
```diff
@@ -123,6 +123,16 @@
   [1,0,1,0,1,0,1,0,0,0,1,1,1,0,0,1,0,1,0,1,0,1,1,1],
 ]
 
+HEART = [
+  [0,4,4,0,4,4,0],
+  [4,2,2,4,2,2,4],
+  [4,2,2,2,2,2,4],
+  [4,0,2,2,2,2,4],
```
Comment (gelstudios) [inline gitfiti.py]: whats with the zero at index 1?

**Comment by gelstudios** [comment]
your heart image has a hole in it, is that intentional?

**Comment by x4m3** [comment]
it is. i am trying to reproduce this
http://moziru.com/images/zelda-clipart-8-bit-8.jpg

if you don't like it and you want to get a full heart, that's fine.

**Comment by Kaligule** [comment]
I see your point, but would like the full hart better. The shiny effect doesn't come out enough with just 7 pixels.

**Comment by x4m3** [comment]
alright , i will update my patch .

**Comment by x4m3** [comment]
good to merge now ?

**Comment by x4m3** [comment]
ready to merge @gelstudios ? although i see some conflicts in the file...

**Comment by gelstudios** [comment]
this is merged but GH wont detect it, closing

**Pull request closed without merging**